### ISSUE TITLE
detriment tutorial segfaults

### ISSUE THREAD

**Issue by teamkill3d** [vinegarhq/sober#250]
### Acknowledgement of checking existent issues

- [X] I have checked if my issue has already been reported, and I am certain that my problem has not already been addressed.

### What problem did you encounter?

if you enter the tutorial of the roblox game detriment, sober segfaults

couldn't be simpler than that

here's the dmesg log:
```
[ 4747.331741] sober[4965]: segfault at f0 ip 0000774d04ac6dc0 sp 0000774d003fc448 error 4 in libnvidia-glcore.so.535.183.01[774d034f9000+269e000] likely on CPU 7 (core 1, socket 0)
[ 4747.331755] Code: 41 0f 95 c1 45 21 e9 4d 85 f6 0f 84 a9 03 00 00 4d 8b 18 bf ff ff ff ff 4d 85 d2 74 0d 49 8b ba f8 00 00 00 8b bf 90 01 00 00 <41> 83 bb f0 00 00 00 00 74 07 41 0b bb 18 01 00 00 4c 8b 9b 40 01
```

### Roblox Experience link

https://www.roblox.com/games/6958681991

**Comment by teamkill3d** [comment]
closed as this is a clone of issue [#244](https://github.com/vinegarhq/sober/issues/244)

**Comment by Vlad55432** [comment]
535, daym
